test/jdk/java/awt/Choice/ChoicePopupLocation/ChoicePopupLocation.java

```diff
@@ -120,6 +125,11 @@ private static void openPopup(final Choice choice) throws Exception {
         robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
         robot.waitForIdle();
         if (choice.getSelectedIndex() == 0) {
+            GraphicsConfiguration ge = GraphicsEnvironment
```

Comment:
`ge` is unused in the latest version of the patch?

Reply:
My mistake. This was a remnant from when I was double-checking that the image would still produce correctly. But yes, I'm just using the existing instance of `ge` since that is determining the screen where the popup is shown.